package.json

```diff
@@ -2,13 +2,14 @@
   "name": "hexo-theme-doc-seed",
   "version": "0.1.0",
   "scripts": {
-    "build": "hexo deploy",
```

Comment:
Feel so-so about this. On one hand it really means 'build'. OTOH they named their command `deploy` (it does no such thing). Either don't change this, _or_ make sure you change the corresponding line in `.travis.yml` or otherwise we break CD.

Reply:
hmm ok, lets just keep as-is. I'll change and update this PR.